lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/IdentityMapFunction.cs

```diff
@@ -26,7 +28,7 @@ namespace Org.Apache.REEF.IMRU.Examples.MapperCount
     public sealed class IdentityMapFunction : IMapFunction<int, int>
     {
         [Inject]
-        private IdentityMapFunction()
+        private IdentityMapFunction(IInputPartition<Stream> dataPartition)
```

Comment:
I just added it to make sure there is a test and example that takes as input IInputPartition instantiated by DataLoadingContext.